src/main/java/com/example/teampandanback/service/ProjectService.java

```diff
@@ -112,4 +115,51 @@ public ProjectCrewResponseDto readCrewList(Long projectId){
                 .build();
     }
 
+    @Transactional
+    public ProjectInvitedResponseDto invitedProject(ProjectInvitedRequestDto projectInvitedRequestDto, SessionUser sessionUser) {
+
+        String decodedString = null;
+        Long decodedLong = null;
+        try {
+            decodedString = aesEncryptor.decrypt(projectInvitedRequestDto.getInviteCode());
+            decodedLong = Long.parseLong(decodedString);
+        } catch (NumberFormatException e) {
+            throw new ApiRequestException("복호화 된 프로젝트ID가 숫자가 아닙니다. 프로젝트ID = " +decodedString);
```

Comment:
프론트 단에 복호화 된 프로젝트 ID가 숫자가 아니라는 내용을 알려줄 필요가 없다고 생각합니다.
```suggestion
            log.info("복호화 된 프로젝트ID가 숫자가 아닙니다. 프로젝트ID = " +decodedString);
            throw new ApiRequestException("유효하지 않은 초대 코드입니다.");
```

Reply:
동의합니다